cauldron/sql.py

```diff
@@ -332,19 +340,32 @@ def select(cls, cur, table: str, order_by: str, columns: list=None, where_keys:
             columns_string = cls._COMMA.join(columns)
             if where_keys:
                 where_clause, values = cls._get_where_clause_with_values(where_keys)
-                query = cls._select_selective_column_with_condition.format(columns_string, table, where_clause,
-                                                                           order_by, limit, offset)
+                if order_by:
+                    query = cls._select_selective_column_with_condition_and_order_by.format(columns_string, table, where_clause,
```

Comment:
There is too much duplication that I see here (`if order_by:` condition everywhere). Best approach would be write a DSL to generate MySQL queries rather than keeping hardcoded strings. However this is definitely beyond the scope of this PR. I will keep that as something to fix in future.